### ISSUE TITLE
Repeatable elements

### ISSUE THREAD

**Issue by JbIPS** [silexlabs/BackNode#23]
I was thinking it would be a good thing to mark some HTML blocs as "repeatable" so the user can duplicate them as many time as he want.
We can use a list for that and just add a new `<li>` with the repeatable content when the user clicks on a + like button.

**Comment by RonanDrouglazet** [comment]
Yeah i have the same idea ! We can put that on all kind of elements ! :) i will work on it soon but if You want to work on this tell me and i switch on an other feature like editable html ;)
